tests/test_nodes.py

```diff
@@ -1880,3 +1882,36 @@ def return_from_match(x):
         inferred = node.inferred()
         assert len(inferred) == 2
         assert [inf.value for inf in inferred] == [10, -1]
+
+
+@pytest.mark.parametrize(
+    "node",
+    list(
+        filter(
+            lambda node: node.__name__
+            not in ["_BaseContainer", "BaseContainer", "NodeNG", "const_factory"],
+            astroid.nodes.ALL_NODE_CLASSES,
+        )
+    ),
```

Comment:
Won't block this merge over it, but stylistically I'd suggest a comprehension instead of listifying a filter.

```suggestion
    [
        node
        for node in astroid.nodes.ALL_NODE_CLASSES
        if node.__name__
        not in ["_BaseContainer", "BaseContainer", "NodeNG", "const_factory"]
    ],
```